app/services/google_map/google_maps_data_builder.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module GoogleMap
+  # Generates metadata for Google Maps with dog name, breed, and location.
+  class GoogleMapsDataBuilder
```

Comment:
Now this works :) For the class name, my suggestions would be to rename it from `GoogleMapsDataBuilder` to `DataBuilder` and use it like `GoogleMap::DataBuilder`

What do you think?

Reply:
Yes this makes sense to me given it is scoped!